doc/content/htgr/httf/lower_plenum_cfd.md

```diff
@@ -0,0 +1,129 @@
+| Melting temperature | $T_{melt}$ | $K$ |
+|  Ring 1    | $\dot{m}$, $T$ | 0.651 $g/s$, 562.2 $K$ |
+|  Ring 2    | $\dot{m}$, $T$ | 4.919 $g/s$, 561.8 $K$ |
+|  Ring 3    | $\dot{m}$, $T$ | 7.423 $g/s$, 541.3 $K$ |
+|  Ring 4    | $\dot{m}$, $T$ | 7.704 $g/s$, 512.0 $K$ |
+|  Ring 5    | $\dot{m}$, $T$ | 2.337 $g/s$, 471.6 $K$ |
+|  LP Roof   | $T$ | 543.48 $K$  |
+|  LP Floor  | $T$ | 494.65 $K$  |
+|  Side Reflectors   | $T$ | 452.37 $K$  |
+|  LP Posts  | $T$ | 526.39 $K$ |
+|  Hot duct  | $T$ | 496.64 $K$ |
+|  Rakes and all other walls   | Adiabatic | N/A  |
+
+The mean LP pressure is 100.5 $kPa$, with which the helium gas has a density of 0.1004 $kg/m^3$. A non-dimensionalization process is performed for the Nek5000 calculations. The reference velocity is 3.47 $m/s$ that is the highest inlet velocity observed for Ring 3 inlet channels. The temperature difference with respect to the side reflector wall temperature (452.37 $K$) is normalized by $\Delta T = 562.20-452.37 = 109.83 (K)$, where the maximum temperature is from Ring 1 inlet channels. During the post-processing, the CFD results can be easily converted back to dimensional quantities for further analyses. 
+
+## Nek5000 Modelling
+
+The reader can find the Nek5000 case files and the associated HTTF lower plenum files (through github LFS) in VTB repository. The mesh information is contained in two files: +httf.re2+ (grid coordiantes, sideset ids, etc.) and +httf.co2+ (mesh cell connectivity). 
+As for the Nek5000 case files, there are +SIZE+, +httf.par+, +httf.usr+, and two auxiliary files +limits.f+ and +utilities.f+ that hosts various post-processing functions, such as solution monitoring, planar avearging, and so forth. 
++SIZE+ is the metafile to specify primary discretization parameters. 
++httf.usr+ is the script where users specify the boundary conditions, turbulence modeling approach and post-processing. 
++httf.par+ is to provide the simulation input parameters, such as material properties, time step size, Reynolds number. 
+
+Now let's dive into the most important case file +httf.usr+. The sideset ids contained in the mesh file are first translated into CFD boundary condition settings in +usrdat2+ block
+
+!listing htgr/httf/lower_plenum_mixing/httf.usr start=subroutine usrdat2() end=subroutine usrdat3() include-end=False
+
+The $k-\tau$ URANS model is specified in the following code block
+
+!listing htgr/httf/lower_plenum_mixing/httf.usr start=subroutine usrdat3() end=subroutine turb_in(wd,tke,tau) include-end=False
+
+The specific inlet temperature and mass flow rates are implemented in +userbc+ with the non-dimensionalized values
+
+!listing htgr/httf/lower_plenum_mixing/httf.usr start=if(id_face.eq.3) end=endif include-end=True
+
+
+## Simulation Results 
+
+The aforementioned CFD cases were carried out on Sawtooth supercomputer located at Idaho National Laboratory. A typical simulation job would use 64 compute nodes with 48 parallel threads per node. 
```

Comment:
```suggestion
The aforementioned CFD cases were carried out on the Sawtooth supercomputer located at Idaho National Laboratory. A typical simulation job would use 64 compute nodes with 48 parallel threads per node. 
```

Reply:
is it actually threads or is it processes?
what command are you using to run this? we should include it?

Reply:
A batch run script is provided for the related CFD cases (on Sawtooth).